tests/components/mealie/snapshots/test_init.ambr

```diff
@@ -1,33 +0,0 @@
-# serializer version: 1
```

Comment:
Did you really mean to delete this file?

Reply:
Yeah, it's unused and happened automatically when running with snapshot-update

Reply:
ah, so I guess snapshot-update is not smart enough to realize I didn't run all tests…